recipes-client/components/recipe-component.tsx

```diff
@@ -1,66 +1,96 @@
 /** @jsx jsx */
 import { Component } from "react";
 import { jsx } from "@emotion/core";
+import FormGroup from "~components/form-group";
 
-function formatTitle(text: string){
-  // Reformat title with first letter uppercase
-  const title = text.replace('_', ' ');
-  return title[0].toUpperCase() + title.slice(1);
-}
+// function formatTitle(text: string){
```

Comment:
let's remove this code rather than comment it out?

Reply:
Agreed!